service/cluster/xtra_db_cluster.go

```diff
@@ -0,0 +1,60 @@
+// dbaas-controller
+// Copyright (C) 2020 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Package cluster TODO
+package cluster
+
+import (
+	"context"
+
+	controllerv1beta1 "github.com/percona-platform/dbaas-api/gen/controller"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// Service implements methods of gRPC server and other business logic.
+type Service struct {
+}
+
+// New returns new Service instance.
+func New() *Service {
+	return new(Service)
+}
+
+// ListXtraDBClusters returns a list of XtraDB clusters.
+func (s *Service) ListXtraDBClusters(ctx context.Context, req *controllerv1beta1.ListXtraDBClustersRequest) (*controllerv1beta1.ListXtraDBClustersResponse, error) {
+	return nil, status.Errorf(codes.Unimplemented, "This method is not implemented yet.")
```

Comment:
@askomorokhov This error should reference i18n key instead of using hard-coded message